src/platform/graphics/CMakeLists.txt

```diff
@@ -38,10 +38,6 @@ add_library(mirplatformgraphicscommon OBJECT
 mir_generate_protocol_wrapper(mirplatformgraphicscommon "zwp_" protocol/linux-dmabuf-unstable-v1.xml)
 
 if (DRM_VERSION VERSION_GREATER 2.4.107)
-  target_compile_definitions(
```

Comment:
Is this correct? We're only checking for `drm >= 2.4.84`, and `drmGetFormatModifierName`  appeared in 2.4.107 as far as I can tell.

Reply:
Hm, and you're not actually removing any of the `#ifdef MIR_HAVE_DRM_GET_MODIFIER_NAME`s?

Reply:
Oops. Dropped that commit as it is messed up